blockdag/dag.go

```diff
@@ -688,6 +639,19 @@ func (node *blockNode) pastUTXOMultiSet(dag *BlockDAG, acceptanceData MultiBlock
 	return ms, nil
 }
 
+func (node *blockNode) selectedParentMultiset(dag *BlockDAG) (*secp256k1.MultiSet, error) {
```

Comment:
From a purist perspective, this is not selectedParentMultiset - in case of genesis it returns an empty multiset, since genesis doesn't have a selectedParent.

For any practical reason this is fine, but I think a comment on top of the function is in order. (And while you are at it, give a complete description of what the function does)